src/js/popup.js

```diff
@@ -675,7 +775,7 @@ $(document).ready(function () {
     $('#fileSelector').change(function () {
         var file = $(this)[0].files[0];
         if (browserType !== 'Edge') {
-            if (file.type !== 'application/json') {
+            if (file.type !== 'application/json' && file.type) {
```

Comment:
I would switch the order here. I don't think it makes a difference except when I read it.
First check if there is a file.type and then check if it doesn't equal json
`if (file.type && file.type !== 'application/json') {`

Reply:
:+1: